python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -80,6 +81,15 @@ def resolve_asset(self, _graphene_info):
         )
 
 
+class GrapheneAssetFreshnessInfo(graphene.ObjectType):
+    currentMinutesLate = graphene.Field(graphene.Float)
```

Comment:
I think these policyDescription field should be required, since it would be populated anytime the `FreshnessInfo` object is returned

so `graphene.Field(graphene.String)` -> `graphene.String`

Reply:
good call (looks like I need it to be `graphene.NonNull(graphene.String)`)

Reply:
yep, that should do it!